### ISSUE TITLE
Step forward, into, back?

### ISSUE THREAD

**Issue by jedahan** [bobthecow/psysh#295]
Is there a way to step forward, and execute the next line after whereami?

**Comment by bobthecow** [comment]
Nope! PsySH is a REPL, and doesn't support (true) breakpoints, catchpoints, stepping, continuing, etc. 

I've long thought it would be great to add support to run PsySH as a remote client for phpdbg, but I haven't had the time or inspiration to implement that yet :)

**Comment by andreiglingeanu** [comment]
@bobthecow There's no way of dropping a break point into the already executing `PHP` code with phpdbh, just like we are able to do with psysh. Good [reading](https://www.reddit.com/r/PHP/comments/1vige3/anybody_use_phpdbg_as_their_debugger_how_does_it/cesp4sk). Do you have some better ideas?

<hr>

[Link](http://stackoverflow.com/questions/28773567/using-phpdbg-with-the-built-in-php-server/38598286#38598286)

**Comment by bobthecow** [comment]
I'm not going to rule this out forever, but PsySH isn't xdebug and isn't trying to be, so I'm closing this (suuuuper old) issue as unplanned :)
